src/loader-components/_ExampleTemplate.jsx

```diff
@@ -27,18 +31,48 @@ const propTypes = {
   description: PropTypes.node,
 
   isExpanded: PropTypes.bool,
+
+  isCssExpanded: PropTypes.bool,
 };
 
 const defaultProps = {
   isExpanded: false,
+  isCssExpanded: false,
 };
 
 const ExampleTemplate = ({
-  example, exampleSrc, title, description, isExpanded,
+  example, exampleSrc, exampleCssSrc, title, description, isExpanded, isCssExpanded,
 }) => {
   const [codeIsVisible, setCodeIsVisible] = useState(isExpanded);
-  const [isBackgroundTransparent, setIsBackgroundTransparent] = useState(false);
+  const [cssIsVisible, setCssIsVisible] = useState(isCssExpanded);
   const theme = React.useContext(ThemeContext);
+  let cssButtonVisible = true;
+  let isCodeSelected = false;
+  let isCssSelected = false;
```

Comment:
What are these for? Should be the using the React useRef hook?

Reply:
If these are just helpers to avoid checking for undefined only once, you can tighten this up:
```suggestion
  const cssButtonVisible = exampleCssSrc !== undefined;
  const isCodeSelected = codeIsVisible;
  const isCssSelected = cssIsVisible;
```
but i'm pretty sure you don't need those last two helpers.

Reply:
Updated `cssButtonVisible` and removed `isCodeSelected` and `isCssSelected`: [2e28aa6](https://github.com/cerner/terra-dev-site/pull/281/commits/2e28aa6167c3cf422dbd74674a56591b9ecbdf9b).